tests/resources/test_project.py

```diff
@@ -0,0 +1,209 @@
+"""
+Tests for fetching project data.
+
+"""
+from unittest import TestCase, mock
+
+from runeq.config import Config
+from runeq.resources.client import GraphClient
+from runeq.resources.project import Project, get_project, get_projects
+
+
+class TestProject(TestCase):
+    """
+    Unit tests for the Project class.
+
+    """
+
+    def setUp(self):
+        """
+        Set up mock graph client for testing.
+
+        """
+        self.mock_client = GraphClient(
+            Config(client_key_id="test", client_access_key="config")
+        )
+
+    def test_attributes(self):
+        """
+        Test attributes for an initialized Project.
+
+        """
+        test_org = Project(
+            id="proj1-id",
+            created_at=1630515986.9949625,
+            updated_at=1630515986.9949625,
+            title="Project 1",
+            description="Test description.",
+            created_by="user-id-1",
+            updated_by="user-id-1",
+            started_at=1630515986.9949625,
+            status="ACTIVE",
+            type="SANDBOX",
+        )
+
+        self.assertEqual("proj1-id", test_org.id)
+        self.assertEqual(1630515986.9949625, test_org.created_at)
+        self.assertEqual("Project 1", test_org.title)
```

Comment:
Can you assert all the required fields here for full coverage

Reply:
sure!